posthog/utils.py

```diff
@@ -320,29 +320,20 @@ def get_js_url(request: HttpRequest) -> str:
     return settings.JS_URL
 
 
-def render_template(
-    template_name: str,
+def get_context_for_template(
```

Comment:
pulled the context generation into its own method so i could test the generated value directly